plugins/fusion/covert.py

```diff
@@ -46,15 +49,17 @@ def is_tor_port(port):
 class CovertSubmitter(PrintError):
     stopping = False
 
-    def __init__(self, dest_addr, dest_port, ssl, tor_port, scheduler, num_slots, num_spares, connect_timeout, submit_timeout):
+    def __init__(self, dest_addr, dest_port, ssl, tor_host, tor_port, scheduler, num_slots, num_spares, connect_timeout, submit_timeout):
         self.dest_addr = dest_addr
         self.dest_port = dest_port
         self.ssl = ssl
 
+        tor_host = tor_host if tor_host else '127.0.0.1'
+
         if tor_port is None:
```

Comment:
to be consistent with fusion.py line350 change below, this ought to be be changed to:
`if tor_host is None or tor_port is None:`